src/alert/classes.scss

```diff
@@ -4,10 +4,30 @@
 
 .iui-alert {
   @include iui-alert;
+}
+
+.iui-alert-icon {
+  @include iui-alert-icon;
+}
+
+.iui-alert-message {
+  @include iui-alert-message;
+}
+
+.iui-alert-link {
+  @include iui-alert-link;
+}
+
+.iui-alert-button {
+  @include iui-alert-button;
+}
+
+.iui-alert-button-icon {
+  @include iui-alert-button-icon;
+}
 
-  @each $status in positive, negative, warning, informational {
-    &.iui-#{$status} {
-      @include iui-alert-category($statusColor: if($status == informational, 'primary', $status));
-    }
+@each $status in positive, negative, warning, informational {
```

Comment:
I moved this outside of the `.iui-alert` selector to reduce specificity. The selectors that `iui-status` targets (`iui-alert-icon`, `iui-alert-message`, `iui-alert-link`, and `iui-alert-button`) are understood to be part of `iui-alert` anyway.

Reply:
Not sure I like it.. It is still risky.
Checking the generated selectors shows that if there is any parent element of alert with any of status classes, alert would get the styling according to that.. 
Maybe instead of having `iui-alert iui-positive`, we could think about having `iui-alert-positive` ?
It is not that we really need to support `iui-alert` just by itself as currently its looking pretty empty (color-wise).. Even though it gives ability to customize it.

Reply:
I'll give that a shot

Reply:
This has been changed